frontend/src/components/top-navbar.tsx

```diff
@@ -23,11 +20,11 @@ export const TopNavBar: FCWOC<{ className?: string }> = ({ children, className }
                         {!isMobile && <BannersBar />}
                         {children}
                         <Box gap="xlarge">
-                            {!isMobile && <a href="/studies" css={{ color: 'white', textDecoration: 'none', fontWeight: 'bold' }}>Studies</a>}
+                            {!isMobile && <Link to="/studies" css={{ color: 'white', textDecoration: 'none', fontWeight: 'bold' }}>Studies</Link>}
                             <Menu alignEnd >
-                                {isMobile && <li><a className="dropdown-item" href="/studies">Studies</a></li>}
-                                <li><a className="dropdown-item" href="/account">My account</a></li>
-                                <li><a className="dropdown-item" href={env.logoutURL} onClick={onLogout}>Log out</a></li>
+                                {isMobile && <li><Link className="dropdown-item" to="/studies">Studies</Link></li>}
+                                <li><Link className="dropdown-item" to="/account">My account</Link></li>
+                                <li><a className="dropdown-item" href={env.logoutURL} onClick={() => user.logout()}>Log out</a></li>
                             </Menu>
```

Comment:
I think this will setup a race between the navigation to `logoutURL`  cancelling any running JS, and the logout() call sending the fetch request to the backend.  To work around that you can make the logout method be bound to the user instance and call it directly here: `onClick={user.logout}`

then have the user method it look like:
```ts
  logout: (ev?: React.MouseEvent<HTMLAnchorElement>) => { 
   if (!ENV.IS_DEV) return

   ev?.preventDefault() 
   .. rest of method
 }
```